src/components/AppLayout/Header/components/WalletIcon/icons/index.ts

```diff
@@ -69,6 +75,10 @@ const WALLET_ICONS: WalletObjectsProps<IconValue> = {
     src: coinbaseIcon,
     height: 25,
   },
+  [WALLET_PROVIDER.SQUARELINK]: {
+    src: squarelinkIcon,
+    height: 25,
+  },
 }
```

Comment:
Please add a type for this object:
```
{ [key: WALLET_PROVIDER]: { src: string, height: number } }
```

Reply:
It's already typed https://github.com/gnosis/safe-react/blob/6efebedce448390f51704915b9284f4fe96fcdf8/src/components/AppLayout/Header/components/WalletIcon/icons/index.ts#L18

Reply:
It's good that's already typed but can be removed because `WalletProviderNames` is the same as just `WALLET_PROVIDER`.